perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -265,16 +265,18 @@ sub meta_line {
 };
 
 sub problem_state_processed {
-    my $self = shift;
+    my ( $self, $c ) = @_;
     return 'fixed - user' if $self->mark_fixed;
     return 'confirmed' if $self->mark_open;
-    return $self->problem_state;
+    my $cobrand_state = $c->cobrand->call_hook(problem_state_processed => $self);
```

Comment:
Doesn't matter/could be later tidying, but just a note that you can access the current cobrand in any DB function call within a web request (and some cron) in `$self->result_source->schema->cobrand`, no need to pass around `$c`.